catroid/src/org/catrobat/catroid/utils/Utils.java

```diff
@@ -114,6 +116,30 @@ public void onClick(DialogInterface dialog, int which) {
 		return true;
 	}
 
+	public static boolean checkIfCrashRecoveryAndFinishActivity(final Activity context) {
+		SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
+		if (preferences.getBoolean(BaseExceptionHandler.RECOVERED_FROM_CRASH, false)) {
+
+			if (!(context instanceof MainMenuActivity)) {
+				context.finish();
```

Comment:
@84n4n4, why do we have to call finish? Does killing the app and it's process not restart the app in the MainMenuActivity?

cc @nishantt95